examples/automm/kaggle_feedback_prize/README.md

```diff
@@ -0,0 +1,162 @@
+# Using MultiModalPredictor with Text Normalization
+
+Take [Feedback Prize - Predicting Effective Arguments](https://www.kaggle.com/competitions/feedback-prize-effectiveness) as an example to show how text normalization can be helpful.
+
+## 1. Preprocess data with text normalization
+
+Text normalization is the task of mapping non-canonical language, typical of speech transcription and computer-mediated communication, to a standardized writing. 
+It is an up-stream task necessary to enable the subsequent direct employment of standard natural language processing tools and indispensable for languages such as Swiss German, 
+with strong regional variation and no written standard. 
+Even though the competition dataset is composed of English only, we found that applying text normalization can reduce `log loss`, the metrics that this competition is evaluating on. 
```

Comment:
We can later try to identify a few examples that pre-/post-normalized text are really different.

Reply:
Good idea. We can add that.

Reply:
I added a couple of examples